rita/src/rita_exit/traffic_watcher/mod.rs

```diff
@@ -151,8 +152,15 @@ pub fn watch(mut babel: Box<Babel>, clients: Vec<Identity>) -> Result<(), Error>
 
 #[cfg(test)]
 mod tests {
+    extern crate env_logger;
+
+    use super::*;
+
     #[test]
-    fn it_works() {
-        assert_eq!(2 + 2, 4);
+    #[ignore]
```

Comment:
does this mean the tests aren't actually being run?

Reply:
A test seemed ideal to put this little routine in. A #[] statement works only for the one thing under it, in this case only this one "test".

Reply:
However, one can run it with `cargo test -p rita -- --test <test_name>`